request-handler/pom.xml

```diff
@@ -63,6 +78,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <!--  Note: Normally, this plugin would be run using
+                "mvn compile assembly:single" but the execution
+                clause below binds it to the normal 'package' lifecycle phase
+                -->
+                <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Probably don't need to use the shade plugin if we're creating a deployment package?  Where there any advantages to using one of these?  I wasn't sure.